### PR TITLE
stripe and general payment flow

### PR DESCRIPTION
**Pull request by stasbar**
add paypal annually codes 
add stripe payment flow 
improve success and cancel pages 
replace NODE_ENV with REACT_APP_ENV since the app is not running in NODE environment

Took 9 hours 4 minutes

### CODE REVIEW

File: src/pages/payment/Payment.tsx [hunk 1]
```diff
@@ -1,50 +1,96 @@
...
+      const err = await stripe.redirectToCheckout({
+        items: [item],
+        customerEmail: userEmail,
+        successUrl: `${window.location.origin}/payment/success?session_id={CHECKOUT_SESSION_ID}`,
+        cancelUrl: `${window.location.origin}/payment/cancel`,
+      });
+
+      setProcessingPayment(false);
+      console.error(err);
+      message.error(err.error.message);
```
Comment (Argeath) [inline src/pages/payment/Payment.tsx:90]: is this message safe to display to users? I mean if it's user-friendly

File: src/places/user.places.ts [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 export const getCurrentUserProfileUrl = (): string => `/user/profile`;
+export const getPaymentUrl = (): string => `/payment`;
 export const getCurrentUserEditProfileUrl = (): string => `/user/wizard`;
 export const getUserProfileUrl = (userId: string): string => `/user/profile/${userId}`;
 export const getUserLoginUrl = (): string => `/user/login`;
 
 export const getCancelSubscriptionUrl = (): string =>
   `https://www.${
-    NODE_ENV === 'development' ? 'sandbox.' : ''
+    REACT_APP_ENV === 'dev' ? 'sandbox.' : ''
```
Comment (Argeath) [inline src/places/user.places.ts:9]: I think it should be `REACT_APP_ENV !== 'prod'`

File: src/pages/payment/Payment.tsx (hunk 1 above)
Comment (stasbar) [reply src/pages/payment/Payment.tsx:90]: Probably not, but for now, it's "nice to have" fix.

File: src/places/user.places.ts (hunk 2 above)
Comment (stasbar) [reply src/places/user.places.ts:9]: Right

**Pull request merged**